packages/text-input/index.js

```diff
@@ -1,42 +1,48 @@
+import classNames from 'classnames'
 import { v1 as uuidv1 } from 'uuid'
+import s from './style.module.css'
 
-function TextInput(props) {
-  //  `field` and `form` props come from Formik
-  //  https://jaredpalmer.com/formik/docs/api/field
-  const { field, form, type, label, placeholder, theme } = props
+function TextInput({
+  className,
+  field,
+  form,
+  type = 'text',
+  label,
+  placeholder,
+  theme = {
+    background: 'light',
+  },
+}) {
+  //  Note: `field` and `form` props typically come from Formik
+  //  ref: https://jaredpalmer.com/formik/docs/api/field
+  //  However, they can also be provided in non-Formik use cases
+  //  and this component will still function as expected.
```

Comment:
Added this note to try to help clarify that it's totally fine to use this component without Formik.